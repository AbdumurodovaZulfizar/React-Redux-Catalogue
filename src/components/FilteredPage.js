import React from "react";
import Categories from "../containers/FilterMethod";
import Foods from "../containers/foods";
import FoodItem from "../containers/Food";

const FilteredPage = () => {
  return (
    <div className="">
        <div className="header-nav"></div>
        <div className="pt-5 home_2">
          <div className="container-fluid row">
          <Categories />
          <Foods />
          <FoodItem />
          </div>
        </div>
    </div>
  )
}

export default FilteredPage;
