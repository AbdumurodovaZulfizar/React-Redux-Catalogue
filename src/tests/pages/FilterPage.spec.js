import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import FilteredPage from '../../components/FilteredPage';
import store from '../../store';

test('should render filterpage page correctly', () => {
  const component = renderer.create(
    <BrowserRouter>
      <Provider store={store}>
        <FilteredPage />
      </Provider>
    </BrowserRouter>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
