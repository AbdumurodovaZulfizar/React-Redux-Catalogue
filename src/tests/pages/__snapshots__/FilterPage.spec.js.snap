// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render filterpage page correctly 1`] = `
<div
  className=""
>
  <div
    className="header-nav"
  />
  <div
    className="home_2"
  >
    <div
      className="container-fluid row m-0 px-0 p-md-3"
    >
      <div
        className=""
      />
      <div>
        <div
          className="col-12 col-md-6 mx-auto mb-4"
        >
          <select
            className="form-select"
            onChange={[Function]}
          />
        </div>
        <div
          className="row"
        />
      </div>
    </div>
  </div>
</div>
`;
